hack/jenkins/e2e.sh

```diff
@@ -688,8 +688,8 @@ case ${JOB_NAME} in
           ${GKE_DEFAULT_SKIP_TESTS[@]:+${GKE_DEFAULT_SKIP_TESTS[@]}} \
           ${REBOOT_SKIP_TESTS[@]:+${REBOOT_SKIP_TESTS[@]}} \
           ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
-          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
-          ) --ginkgo.focus=Reboot"}
+          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}}
```

Comment:
should we keep the \ at the end of this line? (not sure if it matters, but that's the pattern other builds follow)